components/tidb_query_expr/src/impl_json.rs

```diff
@@ -204,6 +204,35 @@ fn quote(bytes: BytesRef) -> Result<Option<Bytes>> {
     Ok(Some(result))
 }
 
+#[rpn_fn(nullable, raw_varg, min_args = 1, max_args = 1)]
+#[inline]
+fn json_valid(args: &[ScalarValueRef]) -> Result<Option<Int>> {
+    assert!(!args.is_empty() && args.len() == 1);
+    let received_et = args[0].eval_type();
+    let r = match received_et {
```

Comment:
If it's always `Ok`, we could return an `Option` in this match, and return the `Ok(r)` at the end of this function :thinking: